go.mod

```diff
@@ -115,7 +115,7 @@ require (
 	sigs.k8s.io/cluster-api-provider-aws v0.0.0
 	sigs.k8s.io/cluster-api-provider-azure v0.0.0
 	sigs.k8s.io/cluster-api-provider-openstack v0.0.0
-	sigs.k8s.io/controller-tools v0.3.1-0.20200617211605-651903477185
+	sigs.k8s.io/controller-tools v0.4.1
```

Comment:
We cannot update controller-tools to v0.4.1, without figuring out what to do about the `IPNet` struct. You can see why if you try to run `./hack/verify-codegen.sh`. You will need to add a replace for this.